### PR TITLE
[PID/(Q)EAA Issuance - Detailed Flow] Discovery

### LINKED ISSUE #263: [PID/(Q)EAA Issuance - Detailed Flow] Discovery
The discovery phase is described as "Steps 1-4 (Discovery): The User selects the PID/(Q)EAA Provider using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance".

To perform this action there's a need of a Trusted List/Registry of Authorized/Accredited PID/(Q)EAA Providers maintained by a supervisory body/authority, that the instance will need to obtain at this purpose. This Trusted List/Registry will need to include related federation endpoint that the wallet will need to access.

There's also, for cross-border service access, the possibility that a user will need  PID/(Q)EAA from a specific Member State, thus the Trusted List/Registry will need to keep this information as well.

### PR DESCRIPTION
**Pull request by SaraConsoliACN**
This commit aims to resolve the issue https://github.com/italia/eudi-wallet-it-docs/issues/263 adding the "trust-evaluation-mechanism" section reference  in chapter Detailed Flow - Step 1-4.

### CODE REVIEW

**Review by peppelinux — APPROVED** [review]

File: docs/en/pid-eaa-issuance.rst [hunk 1]
```diff
@@ -93,7 +93,8 @@ In this section a *Wallet Initiated Flow* is outlined, where the User receives t
     PID/(Q)EAA Issuance - Detailed flow
 
 
-**Steps 1-4 (Discovery):** The User selects the PID/(Q)EAA Provider using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance.
+**Steps 1-4 (Discovery):** The User selects the PID/(Q)EAA Provider using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance as defined in 
+`Trust Evaluation Mechanism <https://italia.github.io/eudi-wallet-it-docs/versione-corrente/en/trust.html#trust-evaluation-mechanism>`_ section of this specification.
```
Comment (grausof) [inline docs/en/pid-eaa-issuance.rst]: what do you think about using a relative path?

Comment (peppelinux) [inline docs/en/pid-eaa-issuance.rst]:
```suggestion
`Trust Model section`_ Trust Evaluation Mechanism, of this specification.
```

Comment (peppelinux) [reply docs/en/pid-eaa-issuance.rst]: @grausof wdyt^

Comment (peppelinux) [reply docs/en/pid-eaa-issuance.rst]: I did it @grausof ^

File: docs/en/pid-eaa-issuance.rst [hunk 2]
```diff
@@ -93,7 +93,8 @@ In this section a *Wallet Initiated Flow* is outlined, where the User receives t
     PID/(Q)EAA Issuance - Detailed flow
 
 
-**Steps 1-4 (Discovery):** The User selects the PID/(Q)EAA Provider using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance.
+**Steps 1-4 (Discovery):** The User selects the PID/(Q)EAA Provider, only among the ones present into the appropriate trust lists, using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance as defined in
```
Comment (peppelinux) [inline docs/en/pid-eaa-issuance.rst:96]:
```suggestion
**Steps 1-4 (Discovery):** The User selects the PID/(Q)EAA Provider, only among the ones present into the appropriate trust lists, using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance as defined in 
```

File: docs/en/pid-eaa-issuance.rst [hunk 3]
```diff
@@ -93,7 +93,8 @@ In this section a *Wallet Initiated Flow* is outlined, where the User receives t
     PID/(Q)EAA Issuance - Detailed flow
 
 
-**Steps 1-4 (Discovery):** The User selects the PID/(Q)EAA Provider using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance.
+**Steps 1-4 (Discovery):** The User selects the PID/(Q)EAA Provider, only among the ones present into the appropriate trust lists, using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance as defined in 
+`Trust Model section`_ Trust Evaluation Mechanism, of this specification.
```
Comment (peppelinux) [inline docs/en/pid-eaa-issuance.rst]:
```suggestion
**Steps 1-4 (Discovery):** The User, using the Wallet Instance, selects the PID/(Q)EAA Provider from those listed in the list of trustworthy entities. The Wallet Instance then processes the Metadata for the selected PID/(Q)EAA Provider as defined in the `Trust Model section`_ Trust Evaluation Mechanism of this specification.
```

**Review by grausof — APPROVED** [review]

**Review by fmarino-ipzs — APPROVED** [review]

**Pull request merged**